doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,845 @@
+
+      * The sections used by the event manager and stopped using orphaned sections.
+
+        * Event manager no longer uses orphaned sections to store information about event types, listeners, and subscribers.    Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+
+      * Reworked priorities.
+
+
+    * Removed:
+
+      *  Forced alignment for x86.
+
+  * :ref:`event_manager_profiler_tracer`:
+
+    * Updated:
+
+      * Event manager Profiler Tracer to no longer use orphaned sections to store profiler information.
+        Hence, the :kconfig:`CONFIG_LINKER_ORPHAN_SECTION_PLACE` option is no longer required in the config file.
+
+TFM libraries
+-------------
+
+* Updated:
+
+  * :ref:`lib_tfm_ioctl_api`:
+
+    * Updated the documentation about the IOCTL request types that have been moved to the upstream TF-M platform support.
+
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nfxlib documentation <nrfxlib:README>` for additional information.
+
+* Added:
+
+  * :ref:`nrfxlib:nrf_dm` and corresponding overview, usage, changelog, and API documentation.
+
+* Updated:
+
+  * :ref:`softdevice_controller`:
+
+    * Updated:
+
+      * :ref:`softdevice_controller_scheduling` by updating :ref:`scheduling_priorities_table` and adding a section on Timing when synchronized to a periodic advertiser.
+
+  * :ref:`nrf_modem`:
+
+    * Updated:
+
+      * Architecture diagram in :ref:`nrfxlib:architecture`.
+      * :ref:`nrfxlib:nrf_modem_delta_dfu`.
+
+    * Removed:
+
+      * Overview diagram in :ref:`nrfxlib:nrf_modem`.
+      * AT socket page.
+      * Extensions page.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+Unity
+-----
+
+* Fixed a bug that resulted in the mocks for some functions not having the ``__wrap_`` prefix. This happened for functions declared with whitespaces between identifier and parameter list.
+
+HID Configurator for nRF Desktop
+--------------------------------
+
+* Updated:
+
+  * The script to recognize the bootloader variant as a DFU module variant for the configuration channel communication. The new implementation is backward compatible - the new version of the script checks for module name and acts accordingly. See the Performing DFU section in :ref:`nrf_desktop_config_channel_script` documentation.
+
+  * Documentation on setting up the dependencies for Debian/Ubuntu/Linux Mint operating system.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``680ed07``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* |no_changes_yet_note|
+
+Mcumgr
```

Comment:
@nvlsianpu and @de-nordic , please check this section.

Reply:
* mcumgr module has been meld into the zephyr-rtos codebase

Reply:
Please delete the whole mcumgr section